packages/app/src/lib/import.ts

```diff
@@ -247,33 +251,37 @@ export async function asLastPass(data: string): Promise<VaultItem[]> {
     return Promise.all(items);
 }
 
-/**
- * Checks if a given string represents a LastPass CSV file
- */
-export function isLastPass(data: string): boolean {
+export async function isLastPass(file: File): Promise<boolean> {
     try {
```

Comment:
Perhaps in the future we can check the mime type and extension of the file first before reading the file. For example in this case, if the mime type is anything other than `text/plain` or `text/csv`, we can safely discard this option. This can wait until we add more import formats though.